active/001-tabular-msa.md

```diff
@@ -0,0 +1,82 @@
+---
+Feature name: tabular-msa
+Start date: 2015-08-14
+RFC PR: <leave empty>
+scikit-bio issue: <leave empty>
+---
+
+# Summary
+
+Replace the current experimental `skbio.alignment.Alignment` class with a
+`TabularMSA` class, providing an API for manipulating tabular (row-column)
```

Comment:
I think we have a pretty reasonable argument against `matrix` at this point. We should update the RFC to include `MatrixMSA` in the alternatives.